src/ace1_compat.jl

```diff
@@ -1,390 +1,276 @@
-# This module is a translation of most ACE1x options to the new ACE 
-# kernels. It is used to provide compatibility with the old ACE1 / ACE1x. 
-
 module ACE1compat
 
 using NamedTupleTools, StaticArrays, Unitful
-import ACEpotentials: DefaultHypers, Models 
-
-using ACEpotentials.Models: agnesi_transform, 
-                            SplineRnlrzzBasis, 
-                            ace_learnable_Rnlrzz
-
-ace1_defaults() = deepcopy(_kw_defaults)
-
-const _kw_defaults = Dict(:elements => nothing,
-                          :order => nothing,
-                          :totaldegree => nothing,
-                          :wL => 1.5,
-                          #
-                          :rin => 0.0,
-                          :r0 => :bondlen,
-                          :rcut => (:bondlen, 2.5),
-                          :transform => (:agnesi, 2, 4),
-                          :envelope => (:x, 2, 2),
-                          :rbasis => :legendre,
-                          #
-                          :pure2b => false,   # TODO: ALLOW TRUE!!!
-                          :delete2b => false, #       here too
-                          :pure => false,
-                          #
-                          :pair_rin => :rin,
-                          :pair_rcut => :rcut,
-                          :pair_degree => :totaldegree,
-                          :pair_transform => (:agnesi, 1, 3),
-                          :pair_basis => :legendre,
-                          :pair_envelope => (:r, 2),
-                          #
-                          :Eref => missing,
-                          #
-                          :variable_cutoffs => false,
-                          )
-
-const _kw_aliases = Dict( :N => :order,
-                          :species => :elements,
-                          :trans => :transform,
-                        )
+import ACEpotentials: Models 
+
+using ACEpotentials.Models: ace_learnable_Rnlrzz
+
+const _kw_defaults = Dict(
```

Comment:
can you please revert these changes? you've removed some kwargs that I want to keep. And I also want to keep my comments.

Reply:
The default arguments are chosen to be compatible with ACE1. This is very much deliberate. There seem to be quite a few changes that you had to make following on from this that we cannot merge I'm afraid. I think we need to discuss this asap.